### DISCUSSION TITLE
Custom plots

### DISCUSSION THREAD

**Discussion by fizixmastr** [MPh-py/MPh#70] [General]
While being able to script models in python is a great new tool for me, and I appreciate that you have made this possible, my initial draw to find a Python-Comsol interface was to have more tools when plotting. Is it possible to use MPh directly for say making the field plot from the capacitor demo rather than just saving the Comsol image?

**Comment by john-hen** [comment]
No, MPh just helps you get the data, you'll have to use some plotting library to visualize it. There are many. That is, if you don't like the image produced by Comsol (which you can of course customize through MPh), then maybe evaluate the data and use something like Matplotlib for 2d and PyVista/VTK for 3d. For the latter, someone (j. miguel zavala-ake) published an [article on Linkedin](https://www.linkedin.com/pulse/integrating-python-comsol-j-miguel-zavala-ak%C3%A9) that may be helpful.
